idd/Energy+.idd.in

```diff
@@ -42533,10 +42533,8 @@ Coil:Cooling:DX:VariableSpeed,
         \type alpha
         \reference CoolingCoilsDXVariableSpeed
    A2,  \field Indoor Air Inlet Node Name
-        \required-field
```

Comment:
The child needs node names, the parent does not since they can be mined from the child.